airdrop-claim-reject-cancel-hts-token-js-sdk/README.md

```diff
@@ -0,0 +1,75 @@
+# Airdrop, Claim, Reject, and Cancel HTS Tokens (HIP-904) with the JS SDK
+
+This example demonstrates how to use the new **HTS Airdrop** feature introduced in **Hedera Improvement Proposal (HIP-904)**. This feature simplifies the process of airdropping both fungible and non-fungible tokens by removing the need for the receiver to pre-associate with a token. It also allows the sender to pay for token associations and supports options for claiming, rejecting, or canceling airdrops.
+
+## How to Run
+
+After cloning the `hedera-code-snippets` repo, navigate to the airdrop example folder:
+
+```
+cd airdrop-claim-reject-cancel-hts-token-js-sdk
+```
+
+Install dependencies:
+
+```
+npm install
+```
+
+Copy the `.env.sample` file and rename it to `.env`:
+
+```
+cp .env.sample .env
+```
+
+Run the aridrop example:
+
+```
+node airdrop.js
+```
+
+### Console Output
+
+![alt text](./img/console-output.png)
```

Comment:
replace image with text in a code block, since the image is simply if console output

Reply:
Addressed.